scripts/install/moviematch.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+_queries() {
+    if [[ ! -f /install/.plex.lock ]]; then
+        echo_warn "Plex server not installed"
+        echo_query "Please enter your Plex Server's address (e.g. domain.ltd:32400)"
+        read -r plexaddress
+    else
+        plexaddress="localhost:32400"
+    fi
+
+    echo_info "Moviematch needs to connect to Plex via a Token. You can hear how to retrieve one here:\nhttps://support.plex.tv/articles/204059436-finding-an-authentication-token-x-plex-token/"
+    echo_query "Please enter your Plex Token"
+    read -r plextoken
+}
+
+_install_deno() {
+
+    #shellcheck source=sources/functions/deno
+    . /etc/swizzin/sources/functions/deno
+    install_deno
+}
+
+mmatchDir="/opt/moviematch"
+
+_install_moviematch() {
+
+    echo_progress_start "Cloning moviematch"
+    git clone https://github.com/LukeChannings/moviematch.git $mmatchDir >> $log
```

Comment:
Should be switched to a release tarball download and an upgrader should be made for that